arch/xtensa/include/esp32/irq.h

```diff
@@ -392,7 +393,7 @@
 #define ESP32_CPUINT_TIMER2         16
 #define ESP32_CPUINT_SOFTWARE1      29
 
-#define ESP32_CPUINT_NINTERNAL      6
+#define ESP32_CPUINT_NINTERNAL      7
```

Comment:
There are only 6 internal interrupts. We don't need to update this macro (it's not used anyway).